### PR TITLE
Display destination address when hovering over a link

### PR DESCRIPTION
**Pull request by barijaona**
Fix issue #1479

### CODE REVIEW

**Comment by barijaona** [comment]
Apparently, when using vertical or horizontal layout, this PR might disturb the appearance of next article when you do "Skip Folder" (through the keyboard shortcut S)

**Comment by barijaona** [comment]
> Apparently, when using vertical or horizontal layout, this PR might disturb the appearance of next article when you do "Skip Folder" (through the keyboard shortcut S)

The issue is not directly linked to this pull request.

**Comment by TAKeanice** [comment]
Instead of going back to the old way of deleting the article html, can't we rather recreate the html or avoid the reload?

**Comment by barijaona** [comment]
> Instead of going back to the old way of deleting the article html, can't we rather recreate the html or avoid the reload?

Just a single html file is kept, and it is now deleted on app exit.

File: Vienna/Sources/Main window/OverlayStatusBar.swift [hunk 1]
```diff
@@ -62,7 +62,7 @@ final class OverlayStatusBar: NSView {
     // MARK: Initialization
 
     init() {
-        super.init(frame: NSRect(x: 0, y: 0, width: 240, height: 26))
+        super.init(frame: NSRect(x: 0, y: 0, width: 4800, height: 26))
```
Comment (Eitot) [inline Vienna/Sources/Main window/OverlayStatusBar.swift:65]: AFAIK, this value is just a default. The sizing happens by Auto Layout. Was there a problem that this addresses?

**Comment by Eitot** [comment]
I am not sure if forcing the overlaying status bar is the right choice. I personally do not use it. Would a new defaults key not be the better solution?

File: Vienna/Sources/Main window/OverlayStatusBar.swift (hunk 1 above)
Comment (barijaona) [reply Vienna/Sources/Main window/OverlayStatusBar.swift:70]: Yes, I add truncated addresses somewhere.

**Comment by barijaona** [comment]
> I am not sure if forcing the overlaying status bar is the right choice. I personally do not use it. Would a new defaults key not be the better solution?

Safari always display link addresses and has no setting to disable them. As far as I know, all browsers do alike: I guess it's for security reasons.
The legacy tests where formerly a technical requirement because link addresses where displayed in the bottom status bar. This is not the case anymore.

File: Vienna/Sources/Main window/OverlayStatusBar.swift (hunk 1 above)
Comment (barijaona) [reply Vienna/Sources/Main window/OverlayStatusBar.swift:70]: I confirm that the problem is in Unified layout.

**Comment by Eitot** [comment]
> Safari always display link addresses and has no setting to disable them. As far as I know, all browsers do alike: I guess it's for security reasons.

In Safari you can disable it under View → Hide Status Bar.

**Comment by barijaona** [comment]
> In Safari you can disable it under View → Hide Status Bar.

Thanks for the remark… I therefore suggest that Vienna keeps Safari's look and feel and avoid adding another preference.

**Comment by barijaona** [comment]
PR rebased and modified according to @Eitot's remarks.

**Pull request merged**